pkg/rancher-desktop/components/Preferences/ApplicationGeneral.vue

```diff
@@ -81,6 +81,7 @@ export default Vue.extend({
       legend-text="Statistics"
     >
       <rd-checkbox
+        data-test="statisticsCheckbox"
```

Comment:
Consider using `data-testid=` and [`getByTestId()`](https://playwright.dev/docs/api/class-locator#locator-get-by-test-id) instead.

Reply:
I would do that in a separate PR that is intended to deal with all our `data-test` locators. Right now no code is using `data-testid`